erizo_controller/erizoClient/src/Room.js

```diff
@@ -53,7 +53,7 @@ const Room = (altIo, altConnectionHelpers, altConnectionManager, specInput) => {
       stream.pc.removeStream(stream);
     }
 
-    log.debug('Removed stream');
+    log.debug(`message: Removed stream, ${stream.toLog()}`);
```

Comment:
I'm wondering if a `toLog()` method would be helpful here - logging `that.roomID`, and maybe other Room level state at some point.